src/main/scala/uk/gov/hmrc/sca/services/WrapperService.scala

```diff
@@ -90,7 +93,8 @@ class WrapperService @Inject()(
         showChildBenefitBanner = showChildBenefitBanner,
         showUserResearchBanner = showUserResearchBanner,
         showAlphaBanner = showAlphaBanner,
-        showBetaBanner = showBetaBanner
+        showBetaBanner = showBetaBanner,
+        optTrustedHelper =  optTrustedHelper
```

Comment:
can we add the same appconfig driven values here like L#71 and L#72, with a default of false, so it can be set in one place if needed and not on every page

Reply:
The parameter isn't expecting true or false, it's waiting for TrustedHelper object to show or not show.

Reply:
Yes true, forgot we're passing a trustedhelper object